### ISSUE TITLE
6 Resistance A-Wings

### ISSUE THREAD

**Issue by iincarna** [Sandrem/FlyCasual#2469]
When I build a squad of 6 resistance A-Wings the application either takes extremely long to start the game or crashes entirely.

**Comment by rune-b** [comment]
Now that's a strange bug. It doesn't happen with other squads?

**Comment by iincarna** [comment]
I believe I encountered the bug with another set of all-like ships, but could not tell you which. I will see if I can reproduce.

**Comment by rune-b** [comment]
I just started a game with 6 resistance A-wings with no problems. Did you use any upgrades?

If it happens again, please take a screenshot of the error message

![Screenshot_20200129-072915](https://user-images.githubusercontent.com/11047987/73333778-26e99e80-426b-11ea-9906-f365694b3af0.png)

**Comment by iincarna** [comment]
Apologies, yes. I should have been clear that it's 6 resistance A-wings with Heroic.

When i get home from work tonight I will see if the omission of Heroic has an impact.

**Comment by rune-b** [comment]
I get no problems with heroic equipped either. Did you play against the AI? If so, do you remember which squad? 

![Screenshot_20200129-163210](https://user-images.githubusercontent.com/11047987/73370807-16f6ac80-42b5-11ea-9013-d1dcb0a89a38.png)

**Comment by iincarna** [comment]
I rarely get the squad to load, but when it does, it's a random AI

**Comment by rune-b** [comment]
I managed to get this error now, twice. On my Windows 10 laptop. Previous attempts were on Android.  Fly Casual 0.9.7.1. The game would stop at the loading screen and after a while windows shows the message that the program has stopped responding. 

List: 6 RZ-2 with Heroic against random AI

But afterwards I was not able to reproduce it. I tried probably 20 times in a row without any problem. Also tried around 10 times on 0.9.7.0 without error.
